apps/financial-remedy/finrem-cos/prod.yaml

```diff
@@ -11,4 +12,6 @@ spec:
         ASSIGNED_TO_JUDGE_DEFAULT_EMAIL: newApplication@justice.gov.uk
         FEATURE_PBA_CASE_TYPE: false
         FEATURE_USE_USER_TOKEN: true
-        VAR_T2: trigger2
\ No newline at end of file
+        VAR_FLUX_V2: trigger3
+      autoscaling:
+        minReplicas: 3
```

Comment:
minReplicas is not something we offer.

it uses `replicas` and `maxReplicas`

https://github.com/hmcts/chart-library/blob/master/library/templates/v2/_hpa.tpl#L11-L12

Reply:
removed that